Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py

```diff
@@ -536,6 +573,8 @@ def list_asset_internet_exposure_command(client: Client, args: dict[str, Any]) -
     name = args.get('name')
     asm_type = args.get('type')
     has_active_external_services = args.get('has_active_external_services')
+    search_from = args.get('search_from', 0)
```

Comment:
The new `search_to` and `search_from` args need to be added to the yml file as well as README